### DISCUSSION TITLE
Desing a badge for plugin

### DISCUSSION THREAD

**Discussion by Zeroto521** [Flow-Launcher/Flow.Launcher#1494] [Ideas]
A flow launcher badge for [Flow.Launcher.Plugin.HelloWorldPython](https://github.com/Flow-Launcher/Flow.Launcher.Plugin.HelloWorldPython) -> [![](https://img.shields.io/badge/flow%20launcher-plugin-blue)](https://github.com/Flow-Launcher/Flow.Launcher.Plugin.HelloWorldPython).

A badge has three main parts which include `logo (left) | label (middle) | message (right)`.
So our badge style could be:
- [![](https://img.shields.io/badge/flow%20launcher-plugin-blue)](https://github.com/Flow-Launcher/Flow.Launcher.Plugin.HelloWorldPython): `logo (NONE) | label (flow launcher) | message (plugin)`, simple one
- `logo (flow launcher) | label (plugin) | message (language or plugin name)`, rich one

> [![](https://img.shields.io/badge/flow%20launcher-plugin-blue)](https://github.com/Flow-Launcher/Flow.Launcher.Plugin.HelloWorldPython) source: `[![](https://img.shields.io/badge/flow%20launcher-plugin-blue)](https://github.com/Flow-Launcher/Flow.Launcher.Plugin.HelloWorldPython)`

Note: logo requires an SVG image.

**Comment by Garulf** [comment]
I actually had an idea similar to this for plugins. But instead of linking to the plugin it would be a `URI` that prompts the user to install the plugin.
